### PR TITLE
chore: fix missing RPM GPG key location

### LINKED ISSUE #41: Key specified in `beaker-server-RedHatEnterpriseLinux.repo` is gone
The key at: https://github.com/beaker-project/beaker-project.org/blob/d291218fa040b9c5b4e59c08894cb6aa70de2c33/yum/beaker-server-RedHatEnterpriseLinux.repo#L6C1-L6C113

In particular: "https://fedoraproject.org/static/352C64E5.txt" no longer exists.

This causes installs to fail. In reality the missing key is the key used by EPEL for RHEL 7. And it could be replaced with the URL: `https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7`

### PR DESCRIPTION
**Pull request by JohnVillalovos**
The previously specified location is no longer accessible. Point at existing location.

Closes: #41

### CODE REVIEW

**Comment by JohnVillalovos** [comment]
@StykMartin Hopefully relatively simple review. Discovered locally when I was trying to build a container.

**Review by StykMartin — APPROVED** [review]

File: yum/beaker-server-RedHatEnterpriseLinux.repo [hunk 1]
```diff
@@ -3,7 +3,7 @@ name=Beaker Server - RedHatEnterpriseLinux$releasever
 baseurl=https://beaker-project.org/yum/server/RedHatEnterpriseLinux$releasever/
 enabled=1
 gpgcheck=1
-gpgkey=https://beaker-project.org/gpg/RPM-GPG-KEY-redhatengsystems https://fedoraproject.org/static/352C64E5.txt
+gpgkey=https://beaker-project.org/gpg/RPM-GPG-KEY-redhatengsystems https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
```
Comment (StykMartin) [inline yum/beaker-server-RedHatEnterpriseLinux.repo:6]: Would be better to use $releasever.. For EPEL 8

Comment (JohnVillalovos) [reply yum/beaker-server-RedHatEnterpriseLinux.repo:6]: Whoops! Didn't see that after you approved and I merged.

I can put a new version up.

Comment (StykMartin) [reply yum/beaker-server-RedHatEnterpriseLinux.repo:6]: We can follow it here - https://github.com/beaker-project/beaker-project.org/issues/43
It is good for now. We still have a long way to go.

**Pull request merged**